cdivdata.cpp

```diff
@@ -51,8 +51,7 @@ BareStatHandler cdiv_not_registered_handler(cdiv_not_registered_oid, &tree);
 BareStatHandler cdiv_no_answer_handler(cdiv_no_answer_oid, &tree);
 BareStatHandler cdiv_not_reachable_handler(cdiv_not_reachable_oid, &tree);
 
-NodeData cdiv_node_data("cdiv_handler",
-                        "6666",
+NodeData cdiv_node_data("sprout",
```

Comment:
Isn't this baking in an assumption that the CDIV AS will always run collocated with Sprout?  We still have the option of running ASs standalone.